lib/atom-inline-blame.js

```diff
@@ -18,24 +20,40 @@ export default {
     }
   },
 
-  activate(state) {
-    this.watchedEditors = [];
+  attachBlamer(editor) {
+    if (!editor) return;
+
+    const { id } = editor;
+    if (!this.watchedEditors.includes(id)) {
+      if (!InlineBlameView) {
+        InlineBlameView = require("./InlineBlameView");
```

Comment:
Why `require` and not `import`? Why not import at the top of the file?

Reply:
Importing a module shouldn't degrade performance afaik. Unless that's not true I would encourage consistency and use an `import` at the top of the file.

Reply:
Loading a module is a synchronous task, and is definitely a hit because it has all the activation code in it. This is loading up Atom with no files and just Timecop open:

<img width="276" alt="import" src="https://user-images.githubusercontent.com/80849/39140463-3e823960-4742-11e8-88ac-2e4def830957.png">

Reply:
The expensive work is done in the constructor not when loading the module. It is very surprising that loading a module as such a significant impact on performance.

Reply:
Correct, but loading the module's code itself is a synchronous task. In the grand scheme of things 100ms is nothing, it only stands out when compared to the other packages. But my goal here is to reduce Atom's startup time which is already ridiculously slow, and this achieves that.